Lib/test/test_minidom.py

```diff
@@ -1560,5 +1562,28 @@ def testProcessingInstructionNameError(self):
         pi = doc.createProcessingInstruction("y", "z")
         pi.nodeValue = "crash"
 
+    def test_minidom_attribute_order(self):
+        xml_str = '<?xml version="1.0" ?><curriculum status="public" company="example"/>'
+        doc = parseString(xml_str)
+        output = io.StringIO()
+
+        with contextlib.redirect_stdout(output) as xml_file:
```

Comment:
`output` and `xml_file` are the same. You can either remove `as xml_file` or replace `output` with `io.StringIO()`.

Reply:
Done.